src/sentry/api/utils.py

```diff
@@ -39,6 +39,48 @@ def default_start_end_dates(now=None, default_stats_period=MAX_STATS_PERIOD):
 
 
 def get_date_range_from_params(params, optional=False, default_stats_period=MAX_STATS_PERIOD):
```

Comment:
use a wrapper function for the old logic which aliases the new parameter to the old one